src/Clustering/View/GMUDefaultClusterRenderer.h

```diff
@@ -159,4 +159,13 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/* Extensions for testing purposes only. */
```

Comment:
Tried to add a different File for this `GMUDefaultClusterRenderer+Testing.h`, but after couple of tries was not able to access the declared methods.

Reply:
Yes have done it, but why it was not coming was unable to figure out. 😕

Reply:
I'm not sure why that might not be working for you. If you share the error you are seeing I can try to help you debug it.

Reply:
PFA Screenshot...
<img width="1434" alt="Screenshot 2020-07-28 at 12 08 40 PM" src="https://user-images.githubusercontent.com/8708394/88628260-469f3980-d0cb-11ea-8345-449484c400d3.png">

Reply:
PFA screenshot.
<img width="1434" alt="Screenshot 2020-07-28 at 12 13 21 PM" src="https://user-images.githubusercontent.com/8708394/88628553-c4fbdb80-d0cb-11ea-80c2-d42bd5402c2e.png">

Reply:
Yes Have done it.

Reply:
What are the target memberships for that file? E.g. `GMUHeatmapTileLater+Testing.h` has UnitTest as its sole target membership.
<img width="1413" alt="Screen Shot 2020-07-28 at 1 09 11 PM" src="https://user-images.githubusercontent.com/463186/88716218-9bdf4900-d0d3-11ea-9c87-637ad4b77b37.png">

Reply:
Please have a look here. Already given it, Can you try it from your end?

<img width="1430" alt="Screenshot 2020-07-29 at 9 34 25 AM" src="https://user-images.githubusercontent.com/8708394/88755402-b3bfd700-d17e-11ea-872e-9977b42e5a92.png">